subtitles/en/csfg_algorithms_race.vtt

```diff
@@ -1,11 +1,15 @@
 WEBVTT
+Kind: captions
+Language: en
 
 NOTE
+
```

Comment:
Is the space needed? In other files there's no space. Is NOTE even needed I wonder